Dockerfile-lite

```diff
@@ -1,65 +1,59 @@
-# Build jbig2enc in a separate stage
-FROM bellsoft/liberica-openjdk-debian:17
+# use alpine
+FROM alpine:latest
 
 ARG VERSION_TAG
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        libreoffice-core \
-        libreoffice-common \
-        libreoffice-writer \
-        libreoffice-calc \
-        libreoffice-impress \
-        unoconv && \
-    rm -rf /var/lib/apt/lists/*
-
-
 # Set Environment Variables
 ENV DOCKER_ENABLE_SECURITY=false \
     HOME=/home/stirlingpdfuser \
     VERSION_TAG=$VERSION_TAG \
     JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -XX:MaxRAMPercentage=75"
-#	PUID=1000 \
+#   PUID=1000 \
 #   PGID=1000 \
 #   UMASK=022 \
 
+# Copy necessary files
+COPY scripts/download-security-jar.sh /scripts/download-security-jar.sh
+COPY scripts/init-without-ocr.sh /scripts/init-without-ocr.sh
+COPY pipeline /pipeline
+COPY src/main/resources/static/fonts/*.ttf /usr/share/fonts/opentype/noto
+COPY src/main/resources/static/fonts/*.otf /usr/share/fonts/opentype/noto
+COPY build/libs/*.jar app.jar
+
+RUN apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        tini \
+        bash \
+        curl \
+        openjdk17-jre \
+# Doc conversion
+        libreoffice-common \
+        libreoffice-writer \
+        libreoffice-calc \
+        libreoffice-impress \
+# python and pip
+        python3 && \
+        wget https://bootstrap.pypa.io/get-pip.py -qO - | python3 - --break-system-packages --no-cache-dir --upgrade && \
+# uno unoconv and HTML
+    pip install --break-system-packages --no-cache-dir --upgrade uno unoconv WeasyPrint && \
```

Comment:
what does `uno unoconv WeasyPrint` do?

Reply:
maybe it should be mentioned in the versions guide

Reply:
Weasyprint is for html to pdf

Reply:
should I keep it?

Reply:
Yes i dont want to lose any functionality

Reply:
I ask because lite didn't included Weasyprint before